lib/mixlib/install/generator/powershell/scripts/helpers.ps1.erb

```diff
@@ -82,10 +82,18 @@ function Get-WebContentOnFullNet {
   
 
   if($bypassList -ne $null){
-     
-	 $bypassList = $bypassList.split(",")
-	 $proxy.BypassList = $byPassList
-  }
+  	$bypassList = $bypassList.split(",")
+    ForEach ($bypassListItem in $bypassList){
+        try {
+          [regex]$bypassListItem | Out-Null
+        }
+        catch {
+          Write-Warning "Wildcard entry $bypassListItem will be ignored, please use regex"
+          Continue
```

Comment:
You could avoid the continue by moving the `$proxy.BypassArrayList.Add($bypassListItem)` into the `try` after the test, but not a blocker.